samples/basic/secret.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import kfp
+from kfp import dsl
+from kfp.gcp import use_gcp_secret
+
+
+def gcs_read_op(url):
+  return dsl.ContainerOp(
+      name='Access GCS using auth token',
+      image='google/cloud-sdk:latest',
+      command=['sh', '-c'],
+      arguments=['gsutil ls ' + str(url)]
```

Comment:
Does `gsutil` work without `gcloud auth activate-service-account`?

BTW, I usually try not to concatenate/inject arguments, but use shell command-line arguments like this:
```
arguments=['gsutil ls "$0"', url],
```

Reply:
Thank you, updated the shell command accordingly. 

Regarding `gcloud auth activate-service-account` use_gcp_secret now adds GOOGLE_APPLICATION_CREDENTIALS by default ([reference](https://github.com/kubeflow/pipelines/blob/dd6e9a82ee85f4358488f1fa886083ab59f56e3d/sdk/python/kfp/gcp.py#L52)).